modules/lang-mustache/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -7,6 +7,5 @@
  */
 
 grant {
-  // needed to do crazy reflection
```

Comment:
I think the entire file can be removed now?

Reply:
+1, I'll make it happen.

Reply:
Done, 0e7ce9d